src/components/dex/MarketPairChart.vue

```diff
@@ -14,32 +14,32 @@
         <div class="day-values">
           <div class="row">
             <div class="label">VOL.</div>
-            <div class="value">{{ tickerData.quoteVolume }}</div>
+            <div class="value">{{ currentTickerData.quoteVolume ? currentTickerData.quoteVolume : 0 }}</div>
           </div>
           <div class="row">
             <div class="label">OPEN</div>
-            <div class="value">{{ $formatNumber(tickerData.open24hr) }}</div>
+            <div class="value">{{ $formatNumber(currentTickerData.open24hr ? currentTickerData.open24hr: 0) }}</div>
```

Comment:
ditto for this and the other places using it here. Perhaps use a computed field to get the various values needed here and avoid the inline ternaries in the interpolation.